src/LibraryManager.Build/RestoreTask.cs

```diff
@@ -16,7 +16,7 @@
 namespace Microsoft.Web.LibraryManager.Build
 {
     public class RestoreTask
-#if NET472
+#if NET481
```

Comment:
`#if NETFRAMEWORK` would be applicable on any legacy version.

Reply:
Updated to just that.